app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -4,122 +4,111 @@ input:-webkit-autofill{
   background-image: none !important;
 }
 
-
 form.block-form {
-  display : inline-block;
+  margin: 20px auto;
 
   label {
-    color : $text-dark-grey;
-  }
-
-  input {
-    &:invalid,
-    &:invalid:required,
-    &:invalid:required:focus {
-      -webkit-box-shadow : none;
-      -moz-box-shadow : none;
-      box-shadow : none;
-      box-shadow : none;
-
-      border : none;
-      color : $text-dark-grey;
-    }
+    color: $text-dark-grey;
+    position: absolute;
```

Comment:
What's the relative container of this? Also, maybe it can be done without `position: absolute`? This property takes the element out of the regular flow of the elements, and sometimes it can lead to nasty problems.